proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -100,3 +107,47 @@ message GenerateBlocksRequest {
 
 message GenerateBlocksResponse {
 }
+
+message GetBalanceRequest {
+}
+message GetBalanceResponse {
+  uint64 confirmed_satoshi = 1;
+  uint64 pending_satoshi = 2;
+}
+
+message Confirmation {
+  uint32 height = 1;
+  google.protobuf.Timestamp timestamp = 2;
+}
+
+message Transaction {
+  cusf.common.v1.ReverseHex txid = 1;
+
+  uint64 fee_satoshi = 2;
+  uint64 received_satoshi = 3;
+  uint64 sent_satoshi = 4;
+
+  Confirmation confirmation_time = 5;
+}
+message ListTransactionsRequest {
+}
+message ListTransactionsResponse {
+  repeated Transaction transactions = 1;
+}
+
+message SendTransactionRequest {
+  // Address -> satoshi amount
+  map<string, uint64> destinations = 1;
+
+  // Fee rate, measured in sat/vbyte. If set to zero, a reasonable
+  // rate is used by asking Core for an estimate.
+  uint64 fee_rate_sat_per_vbyte = 2;
+
+  // if set, the transaction will add a separate OP_RETURN output with this
+  // message.
+  optional string op_return_message = 3;
```

Comment:
This should not be a raw string. That makes it unclear what this really is! Hex-encoded bytes, UTF8, something else entirely? Probably best to use `bytes`, or a new message `Hex` analogous to `ReverseHex`, just not reversed

Reply:
this is true!